docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md

```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
+
+## Download the F3Net Model
+
+To download the pre-trained model or train model by yourself, refer to the 
+[instruction](https://github.com/weijun88/F3Net/blob/master/README.md) in F3Net model repository. 
+To obtain F3Net in ONNX* format you need to put script with next code in `src` folder of model repository and run it:
```

Comment:
`To download the pre-trained model or train the model by yourself, refer to the 
[instruction](https://github.com/weijun88/F3Net/blob/master/README.md) in the F3Net model repository. 
Firstly, the model should be converted to ONNX\* format. Create and run the script with the following content in the `src` directory of the model repository:

Reply:
Done.